### PR TITLE
Issue 1492 plugin csvind

### PR DESCRIPTION
**Pull request by mendoncangelo**
This fixes Soltra/repository#1492

### CODE REVIEW

**Comment by guitarmanvt** [comment]
http://stackoverflow.com/a/181557 @mendoncangelo Python Multiline Conditionals might help you.

File: csvind/views.py [hunk 1]
```diff
@@ -21,6 +21,11 @@ def get_config_namespace(request):
         return request.edgecfg['company_namespace']
     def get_config_alias(request):
         return request.edgecfg['company_alias']
+elif settings.EDGE_VERSION == '2.10' or settings.EDGE_VERSION.startswith('2.10.'):
```
Comment (usrlocalben) [inline csvind/views.py]: please merge this expr with the expr above

File: csvind/xforms.py [hunk 2]
```diff
@@ -5,7 +5,8 @@
 if (   EDGE_VERSION == '2.1' or EDGE_VERSION.startswith('2.1.')
     or EDGE_VERSION == '2.2' or EDGE_VERSION.startswith('2.2.')):
     from .xforms_22 import *
-elif EDGE_VERSION == '2.9' or EDGE_VERSION.startswith('2.9.'):
+elif ( EDGE_VERSION == '2.9' or EDGE_VERSION.startswith('2.9.')
+      or EDGE_VERSION == '2.10' or EDGE_VERSION.startswith('2.10.')):
```
Comment (guitarmanvt) [inline csvind/xforms.py]: PEP8 talks about line breaks around a binary operator here: https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

I like to run `pylint` to find warnings/errors on code changes I'm committing. I think it's a team standard to do so before submitting a Merge Request. Here's what `pylint` said about those two lines:
```
csvind/xforms.py:6:5: W503 line break before binary operator
csvind/xforms.py:9:7: W503 line break before binary operator
```

File: csvind/views.py (hunk 1 above)
Comment (guitarmanvt) [inline csvind/views.py]: @usrlocalben pointed this out in a comment that disappeared. This logical clause is identical to the preceding clause, so the two logical conditions should be merged.

File: csvind/xforms.py (hunk 2 above)
Comment (guitarmanvt) [inline csvind/xforms.py]: @usrlocalben pointed out that the formatting is consistent with older code. I withdraw this suggestion.

**Pull request merged**